src/palace/manager/api/authentication/basic.py

```diff
@@ -223,7 +228,8 @@ class BasicAuthProviderLibrarySettings(AuthProviderLibrarySettings):
             "values here. This value is not used if <em>Library Identifier Restriction Type</em> "
```

Comment:
It might be worth changing the type of this field to `LibraryIdenfitierRestrictionFields` instead of a string, if we're going to enforce an enum of values here.

Reply:
We might not be able do that for all integrations (e.g., with the Millenium integration, since we need to support field names that can vary from library to library). But I'll see what we can do.